async-bytecomp.el

```diff
@@ -91,7 +91,7 @@ All *.elc files are systematically deleted before proceeding."
     (async-start
      `(lambda ()
         (require 'bytecomp)
-        ,(async-inject-variables "\\`\\(load-path\\)\\|byte\\'")
+        ,(async-inject-variables "\\`\\(load-path\\'\\|byte-\\)")
```

Comment:
Shouldn't the group also be shy?

Reply:
Shy? I'm trying to match either:

* start `load-path` end
* start `byte-

Reply:
By shy I mean `\\(?:...\\)` instead of `\\(...\\)`, as I don't see the captured match group being used anywhere, though I could be wrong.

Reply:
Oh, non-capturing. Yeah, might as well